src/components/pages/python.tsx

```diff
@@ -45,10 +51,27 @@ function Python(): JSX.Element {
       { ['print']: Color.Orange },
       { ['(color']: Color.White },
       { [' != ']: Color.Orange },
-      { ['"red")']: Color.White},
+      { ['"red"']: Color.Blue},
+      { [')']: Color.White},
     ],
   ];
 
+  const [dropValues, setDropValues] = useState({
+    top: false,
+    middle: false,
+    bottom: false,
+  });
+
+  const selected = (value:string, pos:Position) => {
+    const val = value === 'True';
+    const newDropValues = dropValues;
+    newDropValues[pos] = val;
```

Comment:
i dont think this is going to copy the array properly. 
```suggestion
    const newDropValues = {...dropvalues, [pos]: val};
```